### PR TITLE
material-icons-ttf: init at 3.0.1

### PR DESCRIPTION
**Pull request by mpcsh**
###### Motivation for this change
There was no package for this icon set, and now there is!

###### Things done
- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] ~~Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))~~ N/A
- [ ] ~~Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`~~ N/A
- [x] ~~Tested execution of all binary files (usually in `./result/bin/`)~~ N/A - but did test that the `.ttf` files end up in the same place as other icon packs (e.g. Font Awesome) and show up after an `fc-cache`
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).
---

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -13034,6 +13034,8 @@ with pkgs;
 
   man-pages = callPackage ../data/documentation/man-pages { };
 
+  material-icons-ttf = callPackage ../data/fonts/material-icons-ttf { };
```
Comment (LnL7) [inline pkgs/top-level/all-packages.nix]: I would drop the ttf suffix here

**Review by LnL7 — APPROVED** [review]

File: pkgs/top-level/all-packages.nix (hunk 1 above)
Comment (rycee) [inline pkgs/top-level/all-packages.nix]: Agreed, I think it should be removed both from the directory and the entry in all-packages.

**Comment by mpcsh** [comment]
Renamed to just `material-icons`

**Review by LnL7 — APPROVED** [review]

**Comment by rycee** [comment]
Thanks for the contribution! Though, for next time please squash the commits.

**Comment by mpcsh** [comment]
@rycee will do, thanks for the heads up

**Pull request merged**